packages/twenty-front/src/pages/settings/ai/forms/components/SettingsAIAgentForm.tsx

```diff
@@ -0,0 +1,201 @@
+import styled from '@emotion/styled';
+import { useLingui } from '@lingui/react/macro';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { useAiModelOptions } from '@/ai/hooks/useAiModelOptions';
+import { IconPicker } from '@/ui/input/components/IconPicker';
+import { Select } from '@/ui/input/components/Select';
+import { TextArea } from '@/ui/input/components/TextArea';
+import { TextInput } from '@/ui/input/components/TextInput';
+import { useGetRolesQuery } from '~/generated-metadata/graphql';
+
+const StyledInputsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(8)};
+`;
+
+const StyledFormSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledIconNameRow = styled.div`
+  align-items: flex-start;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledIconContainer = styled.div`
+  flex-shrink: 0;
```

Comment:
seeing some duplicate code. Not a big deal but this will likely lead to regressions in the future as one form evolve and not the other. We should try to standardize settings as much as possible. At this point, because we have many existing pages it should be unlikely that we have to keep doing this kind of container